build.gradle.kts

```diff
@@ -51,6 +55,13 @@ dependencies {
         because("has methods to simplify the reflection")
     }
 
+    ktlint(libs.logback.core) {
+        because("'1.3.5' version has CVE-2023-6378, CVE-2024-12798, CVE-2024-12801 vulnerabilities")
+    }
+    ktlint(libs.logback.classic) {
+        because("'1.3.5' version has CVE-2023-6378 vulnerability")
+    }
```

Comment:
note: honestly, I don't understand why there would be a need to update dependencies in the linter... It does not go into production or even as a transitive dependency. We can simply exclude `ktlint` configuration from vulnerabilities verification

Reply:
I would like to see existed vulnerabilities